packages/suite/src/components/suite/modals/ReviewTransaction/components/Summary.tsx

```diff
@@ -168,7 +168,12 @@ const Summary = ({
     const theme = useTheme();
     const { symbol } = account;
 
-    const amount = new BigNumber(tx.totalSpent).minus(tx.fee).toString();
+    let amount;
+    if (tx.token) {
```

Comment:
not sure if I can leave it like this, but the point is erc20 tokens have `totalSpent` without fees, but I'll gladly do a better fix if you point me to the right direction. Thx

Reply:
lgtm, but maybe @szymonlesisz will have some idea